scripts/psij-consol.py

```diff
@@ -0,0 +1,95 @@
+#! /usr/bin/python3
+
+import psij
+from psij import JobExecutor
+from psij import JobSpec
+import sys
+import os
+import argparse
+from psij import Import
+
+
+
+
+parser = argparse.ArgumentParser(prog='psij-consol')
+subparser = parser.add_subparsers(dest="command", help='Subcommands')
+validate_parser = subparser.add_parser("validate", help='validate JobSpec file')
+validate_parser.add_argument("file", help="JobSpec file")
+execute_parser = subparser.add_parser("run", help='execute JobSpec file')
+execute_parser.add_argument("file", help="JobSpec file")
```

Comment:
I wonder if it should accept multiple files? The tricky part would then be how that interacts with ``-n``.

Reply:
We can always make this change later though.